jaxrs-api/src/main/java/jakarta/ws/rs/BeanParam.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2012, 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025 Contributors to the Eclipse Foundation
```

Comment:
According to Eclipse Foundation Developer Handbook the Copyright year must be the INITIAL date. You MUST NOT update it.